src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit } from '@angular/core';
+import { NgGridConfig, NgGridItemConfig } from 'angular2-grid';
+import _ from 'lodash';
+
+//Final Object need to be defined.
+interface DotLayoutContainer {
+    id: string;
+    config: NgGridItemConfig;
+}
+
+@Component({
+    selector: 'dot-edit-layout-grid',
+    templateUrl: './dot-edit-layout-grid.component.html',
+    styleUrls: ['./dot-edit-layout-grid.component.scss']
+})
+export class DotEditLayoutGridComponent implements OnInit {
+    public gridContainers: Array<DotLayoutContainer> = [];
+    private static maxColumns: number = 12;
+    private static newRowTemplate: NgGridItemConfig = { fixed: true, sizex: 3, maxCols: 12, maxRows: 1 };
+    private static defaultEmptyGridRows: NgGridItemConfig = {
+        fixed: true,
+        sizex: 12,
+        maxCols: 12,
+        maxRows: 1,
+        col: 1,
+        row: 1
+    };
+    addButton;
+    gridConfig: NgGridConfig = <NgGridConfig>{
```

Comment:
Extra space in ```> {```

Reply:
looks like prettier do not add space for objects just for closures.